core/trino-main/src/test/java/io/trino/operator/TestExchangeClient.java

```diff
@@ -240,8 +240,6 @@ public void testBufferLimit()
 
         long start = System.nanoTime();
 
-        // start fetching pages
-        exchangeClient.scheduleRequestIfNecessary();
```

Comment:
> Decrease visibility of scheduleRequestIfNecessary

commit title is about visibility, while this removes a call site

Reply:
This call site wasn't necessary and was removed to decrease method visibility.